packages/amplication-data-service-generator/src/server/prisma/create-prisma-schema.ts

```diff
@@ -55,7 +55,13 @@ export async function createPrismaSchemaInternal({
     return enumFields.map((field) => createPrismaEnum(field, entity));
   });
 
-  const schema = PrismaSchemaDSL.createSchema(models, enums, dataSource, [
+  const prismaDataSource = {
+    name: PrismaSchemaDSL.DataSourceProvider[dataSource.provider].toLowerCase(),
+    provider: PrismaSchemaDSL.DataSourceProvider[dataSource.provider],
+    url: new PrismaSchemaDSL.DataSourceURLEnv(dataSource.urlEnv),
+  };
+
+  const schema = PrismaSchemaDSL.createSchema(models, enums, prismaDataSource, [
     clientGenerator,
```

Comment:
To completely remove the dependency on prisma-schem-dsl from the plugins, we should also replace this object with an internal type 
```suggestion
    clientGenerator,
```